src/quo2/components/inputs/locked_input/view.cljs

```diff
@@ -0,0 +1,30 @@
+(ns quo2.components.inputs.locked-input.view
+  (:require [react-native.core :as rn]
+            [status-im2.contexts.quo-preview.tabs.segmented-tab]
+            [quo2.foundations.colors :as colors]
+            [status-im.ui.components.icons.icons :as icons]
+            [quo2.components.inputs.locked-input.style :as style]
+            [quo2.theme :as theme]
```

Comment:
please use as `[quo2.theme :as quo.theme]` so that there is no chance of `theme` import clashing with `theme` prop